addons/binding/org.openhab.binding.kostalinverter/src/main/java/org/openhab/binding/internal/kostal/inverter/ThirdGeneration/ThirdGenerationChannelDatatypes.java

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright (c) 2010-2019 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.internal.kostal.inverter.thirdgeneration;
+
+/**
+ * The {@link ThirdGenerationChannelDatatypes} enumeration contains the data types provided by the device
+ *
+ * @author René Stakemeier - Initial contribution
+ */
+enum ThirdGenerationChannelDatatypes {
+    Integer,
```

Comment:
All Enum values should be in upper case: `Integer` -> `INTEGER`.

Reply:
Will be in the next commit